drivers/platform/nintendo3ds/ctr_spi.c

```diff
@@ -30,10 +30,10 @@ struct ctr_spi {
 
 /* CNT register bits */
 #define SPI_CNT_CHIPSELECT(n)	((n) << 6)
-#define SPI_CNT_XFER_READ		(0 << 13)
-#define SPI_CNT_XFER_WRITE		(1 << 13)
-#define SPI_CNT_BUSY			BIT(15)
-#define SPI_CNT_ENABLE			BIT(15)
+#define SPI_CNT_XFER_READ	(0 << 13)
+#define SPI_CNT_XFER_WRITE	(1 << 13)
+#define SPI_CNT_BUSY	BIT(15)
```

Comment:
not a fan of the indentation here, unless there are tabs that aren't visible in github's review UI